packages/leancode_hooks/CHANGELOG.md

```diff
@@ -1,11 +1,14 @@
+# 0.0.3
+
+- Add tests for `useThrottle`
+- Fix broken links in README and CHANGELOG.
+
 # 0.0.2
 
 - Improve README.
 - Delete `useGoogleMapController` hook. You can view it
-  [here](google_map_controller).
+  [here](https://gist.github.com/bartekpacia/3cac94d29f8e6e438e57b366a742f73e).
```

Comment:
We should have a centralized place for all external hooks (readme?)

Reply:
I don't think README is good for this. Some hooks might be long. 

But it's still betther than some gist, I admit.

Reply:
> Some hooks might be long.

I just meant links to them :D

Reply:
Ah, okay :)

Reply:
We have it in Git history, please refrain from using private GitHub account